apps/hyperdrive-trading/src/ui/hyperdrive/longs/OpenLongPreview/OpenLongPreview.tsx

```diff
@@ -153,9 +154,7 @@ export function OpenLongPreview({
       />
       <LabelValue
         label="Matures in"
-        value={`${numDays} days, ${new Date(
-          Date.now() + termLengthMS,
-        ).toLocaleDateString()}`}
+        value={`${numDays} days, ${formatDate(Date.now() + termLengthMS)}`}
```

Comment:
Could be exciting/attention-grabbing to see "Today"

```suggestion
        value={`${numDays ? `${numDays} days` : "Today!"}, ${formatDate(Date.now() + termLengthMS)}`}
```

Idk about the "!", but seemed playful 🙂